### PR TITLE
draft leadership transition post

### PR DESCRIPTION
**Pull request by shannonpileggi**
Here is a first draft, open to all suggestions!

### CODE REVIEW

**Comment by shannonpileggi** [comment]
I didn't see @rivaquiroga and Averi G in the list of github reviewers - should they be added somehow?

File: content/post/2022-11-18-global-leadership-team-transition/index.en.Rmd [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+times, and for continuing to expand upon the foundation first established by Gabriela
+de Queiroz.
+
+We are pleased to share the new members of the Global Leadership Team: Athanasia 
+Monika Mowinckel (Norway), Yanina Bellini Saibene (Argentina), Shannon Pileggi
+(United States), Riva Quiroga (Chile), and Averi Giudicessi (United States). We are
+committed to the R-Ladies Global mission of encouraging, inspiring, and empowering
+people of minority genders in the R community, and we are honored to serve the 
+global organization in this capacity.
+
```
Comment (drmowinckels) [inline content/post/2022-11-18-global-leadership-team-transition/index.en.Rmd:30]: We are not going to be adding any code chunks here, so I'd suggest we delete the Rmd an only edit in the pure md for this post. :)

Comment (shannonpileggi) [reply content/post/2022-11-18-global-leadership-team-transition/index.en.Rmd]: done!

File: content/post/2022-11-18-global-leadership-team-transition/index.en.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+The organization is built upon layers of support - the global leadership team, the broader global team, local chapter organizers, and community members - all who donate their time and effort to sustain the mission of promoting gender diversity in the R community.
+
+Having served on the leadership team since the inception of the global organization in 2016, Erin LeDell (United States), Claudia Vitolo (Italy), and Hannah Frick (England) are now stepping down.
+Over the past six years, R-Ladies experienced tremendous growth in community members and reach around the world.
+In more recent years, R-Ladies survived and thrived throughout the pandemic by pivoting to online meet-ups, enabling knowledge sharing and community engagement both within and across chapters. 
+We are forever grateful to Erin, Claudia, and Hannah for their thoughtful guidance supported by the many volunteers at all levels of the organization.
+
+We are pleased to share the new members of the Global Leadership Team: Athanasia Monika Mowinckel (Norway), Yanina Bellini Saibene (Argentina), Shannon Pileggi (United States), Riva Quiroga (Chile), and Averi Giudicessi (United States).
```
Comment (yabellini) [inline content/post/2022-11-18-global-leadership-team-transition/index.en.md:23]:
```suggestion
We are pleased to share the new members of the Global Leadership Team: Athanasia Monika Mowinckel (Norway), Yanina Bellini Saibene (Argentina), Shannon Pileggi (United States), Riva Quiroga (Chile), and Averi Giudicessi (United States).  We also want to thanks the work of Saranjeet Kaur Bhogal (India) for all her work on the last year for this transition.
```

**Review by yabellini — COMMENTED** [review]
Two small suggestion.  Thanks so much for write this Shannon!

**Review by drmowinckels — APPROVED** [review]

**Review by rivaquiroga — APPROVED** [review]

**Review by yabellini — APPROVED** [review]
Beautiful.  Thanks so much!

**Review by hfrick — APPROVED** [review]

**Review by ledell — APPROVED** [review]
This is great, thank you!

**Review by cvitolo — APPROVED** [review]
Great post, thank you so much!

**Pull request merged**